### DISCUSSION TITLE
Usage documentation

### DISCUSSION THREAD

**Discussion by MusabDisi** [google/filament#7033] [Q&A] [answered]
Is there a source where I can learn how to use this library precisely in Android, I was able to load models and so on but I want to manipulate the models and I cannot find enough resources online and the sample apps are not covering everything, 😕
I need to understand what does each method do and how to connect things together.

**Comment by romainguy** [comment, marked as answer]
All APIs should be documented already, but we are missing indeed high level documentation on how to connect them together. The javadocs should however help you get started.
